### PR TITLE
docs: document rdt related annotations

### PR DESCRIPTION
**Pull request by marquiz**

### CODE REVIEW

File: docs/policy/rdt.md [hunk 1]
```diff
@@ -24,14 +24,30 @@ CLOS  (under `/sys/fs/resctrl`) corresponding the RDT class. CRI-RM will configu
...
 
-By default there is a direct mapping between Pod QoS classes and RDT classes:
-the containers of the Pod get an RDT class with the same name as its QoS class
-(Guaranteed, Burstable or Besteffort). However, that can be overridden with the
-rdtclass.cri-resource-manager.intel.com Pod annotation. You can also specify
-RDT classes other than Guaranteed, Burstable or Besteffort. In this case, the
-Pod can only be assigned to these classes with the Pod annotation, though.
-The default behavior can also be overridden by a policy but currently none of
-the builtin policies do that.
+CRI-RM maingains a direct mapping between Pod QoS classes and RDT classes. If
```
Comment (wpross) [inline docs/policy/rdt.md:27]: maintains

File: docs/policy/rdt.md [hunk 2]
```diff
@@ -24,14 +24,30 @@ CLOS  (under `/sys/fs/resctrl`) corresponding the RDT class. CRI-RM will configu
...
+RDT is enabled CRI-RM tries to assign containers into an RDT class with a name
+matching their Pod QoS class. This default behavior can be overridden with
+pod annotations,
```
Comment (wpross) [inline docs/policy/rdt.md:30]: annotations.

File: docs/policy/rdt.md (hunk 1 above)
Comment (marquiz) [reply docs/policy/rdt.md]: Thx, fixed

**Review by wpross — APPROVED** [review]

**Review by klihub — COMMENTED** [review]
LGTM

**Review by jukkar — APPROVED** [review]

**Pull request merged**